packages/epo-widget-lib/vite.config.ts

```diff
@@ -13,6 +13,30 @@ export default defineConfig({
       formats: ["es", "umd"],
       fileName: (format) => `epo-widget-lib.${format}.js`,
     },
+    rollupOptions: {
```

Comment:
needed to pull in a bunch of these rollup settings so that our dependencies don't get bundled